opencti-platform/opencti-front/src/private/components/settings/users/User.tsx

```diff
@@ -405,6 +411,22 @@ const User: FunctionComponent<UserProps> = ({ userData, refetch }) => {
                 </Typography>
                 {user.lastname || '-'}
               </Grid>
+              <Grid item={true} xs={6}>
+                  <Typography variant="h3" gutterBottom={true}>
+                    {t('Account Status')}
+                  </Typography>
+                  <ItemAccountStatus
+                    account_status={user.account_status}
+                    label={t(user.account_status || 'Unknown')}
```

Comment:
missing account statuses translations in Localization.js

Reply:
@Archidoit - Corrected. Also, did find a few existing 'unknown' translations that were typo'ed with 'unkown', so corrected those.